.travis.yml

```diff
@@ -88,6 +87,7 @@ script:
   - export FFLAGS="-Wall -Wextra -pedantic ${WERROR:+-Werror}" 
   - mkdir build && pushd build &&
     cmake -DCMAKE_PREFIX_PATH="$HOME/kokkos;$HOME/scafacos"
+          -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
I don't think that is needed as we use symlinked compilers.

Reply:
I removed the symlink

Reply:
Ok, this makes it more obvious.